src/arraystring.rs

```diff
@@ -65,8 +76,8 @@ impl<const N: usize> ArrayString<N> {
     /// ```
     #[inline]
     pub fn try_from_str<S>(s: S) -> Result<Self, OutOfBounds>
-    where
-        S: AsRef<str>,
+        where
```

Comment:
Are you using `cargo fmt` to autoformat the code?

Reply:
Might have forgotten to enable it